tests/testthat/_snaps/ptype.md

```diff
@@ -7,3 +7,55 @@
       x Doesn't work on recipes created prior to version 1.1.0.
       i Please recreate recipe.
 
+# recipes_ptype_validate() works
+
+    Code
+      recipes_ptype_validate(rec, data_new)
+    Condition
+      Error:
+      ! Not all variables in the recipe are present in the supplied training set: `id`.
+
+---
+
+    Code
+      recipes_ptype_validate(rec, data_new)
+    Condition
+      Error:
+      ! Not all variables in the recipe are present in the supplied training set: `id` and `x1`.
+
+---
+
+    Code
+      recipes_ptype_validate(rec, data_new)
+    Condition
+      Error:
+      x The following variable has the wrong class:
+      * `id` must have class <numeric> not <integer>.
+
+---
+
+    Code
+      recipes_ptype_validate(rec, data_new)
+    Condition
+      Error:
+      x The following variables has the wrong class:
+      * `id` must have class <numeric> not <integer>.
+      * `x2` must have class <integer> not <factor>.
+      * `y` must have class <numeric> not <integer>.
```

Comment:
For this type check specifically, I do wonder if we may be unintentionally putting users in a place where they didn't actually choose to do any coercion but some function of ours did it under the hood and they can no longer prep/bake. 

Let's keep an eye on extratests once this is merged.

Reply:
🤞 

I really hope we don't, because it would be good if we did 😅